src/frontend/apps/e2e/__tests__/app-impress/doc-search.spec.ts

```diff
@@ -24,91 +25,59 @@ test.describe('Document search', () => {
       page.getByLabel('Search modal').getByText('search'),
     ).toBeVisible();
 
-    await expect(
-      page.getByPlaceholder('Type the name of a document'),
-    ).toBeVisible();
+    const inputSearch = page.getByPlaceholder('Type the name of a document');
+    await inputSearch.click();
+    await inputSearch.fill(doc1Title);
+
+    const listSearch = page.getByRole('listbox').getByRole('group');
+
+    await page
+      .getByPlaceholder('Type the name of a document')
+      .press('ArrowDown');
+
+    const rowdoc = listSearch.getByRole('option').filter({
+      hasText: doc1Title,
+    });
+
+    await expect(rowdoc.getByText('keyboard_return')).toBeVisible();
+    await expect(rowdoc.getByText(/seconds? ago/)).toBeVisible();
   });
 
   test('it checks search for a document', async ({ page, browserName }) => {
-    const id = Math.random().toString(36).substring(7);
-
-    const doc1 = await createDoc(page, `My super ${id} doc`, browserName, 1);
-    await verifyDocName(page, doc1[0]);
-    await page.goto('/');
```

Comment:
I don't think this step should be deleted. Because potentially, this button will not have the same behavior with multi-page for example. Or maybe a future screen. We might as well go back to the place where we are sure that this is the desired behavior. WDYT ?